### PR TITLE
a new executable appears!

### PR DESCRIPTION
**Pull request by kroner**
what do you think, Shane??

### CODE REVIEW

File: trials/death/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+project(death)
```
Comment (scottsha) [inline trials/death/CMakeLists.txt:1]: This is a morbid executable name. I think we should call it "Genocide"

Comment (kroner) [reply trials/death/CMakeLists.txt:1]: :(

File: trials/death/death.cpp [hunk 2]
```diff
@@ -0,0 +1,13 @@
+//
+// Created by robert on 11/21/20.
+//
+#include <iostream>
+
+int main() {
+    for(int i = 0; i < 10; i++) {
```
Comment (scottsha) [inline trials/death/death.cpp:7]: You can initialize a stupid int with auto if you like

**Review by scottsha — APPROVED** [review]

File: trials/death/death.cpp (hunk 2 above)
Comment (kroner) [reply trials/death/death.cpp:7]: do you think that it is better to use "auto"?

Comment (scottsha) [reply trials/death/death.cpp:7]: No. Not in this case. But if you were compariing against a size, which is often type size_t then you get an annoying conversion warning

**Pull request merged**